docs/How-To/cli/README.md

```diff
@@ -57,16 +62,15 @@ Let's create `proposal.json` file. Depending on what budget plan you plan to cre
 ```
 
 ```bash
-# Submit a governance proposal
+# Submit a public ratio plan governance proposal
```

Comment:
```suggestion
# Submit a parameter changes proposal to create a budget plan
```

I think `public ratio plan` is for farming

Reply:
I think I accidentally copied and pasted it. Thanks for the catch!